### ISSUE TITLE
Post/archive the video from the live stream 

### ISSUE THREAD

**Issue by dcwalk** [ournetworks/2018#87]
- [x] Set up youtube acct with managers
- [x] Decide on desired format / resolution output (in comments below)
- [x] Process recorded video, [**tracked below**](https://github.com/ournetworks/2018/issues/87#issuecomment-412313456)
- [x] Locations dec'n (in order):
  1. Youtube
  2. IPFS
  3. Internet Archive
  4. BitTorrent? DTube?
- [x] Have archive pinned elsewhere (@benhylau)
- [x] Update website with links to ipfs archive and posting locations (in https://github.com/ournetworks/ournetworks.ca/pull/78)
- [x] Update youtube with description pointing back to other versions
- [x] Add to internet archive and update website with links (in https://github.com/ournetworks/ournetworks.ca/pull/84)
---

From https://github.com/tomeshnet/ipfs-live-streaming/issues/8#issuecomment-400164966 it appears the decision was:

> The readme of this repo is where all the decisions about the setup is captured. I think where to put Our Networks content should be documented in Our Networks repo. A few things were talked about, including Internet Archive, ipfs, YouTube...

So we should make that decisions as an organizing committee, including timelines

**Comment by dcwalk** [comment]
Processed Videos Mid-august:
- youtube
- archive.org
- IPFS archive
- vimeo (--)

Raw Stream:
- IPFS rewind (secret stretch)

**Comment by dcwalk** [comment]
I think we can reframe this as an issue that focuses on confirming things we uploaded :) (in the interest of fewer new issues)

**Comment by Shrinks99** [comment]
I'm happy to be a manager for the YouTube account (will need to be in order to publish anyhow).  I believe Protocol Labs said they would pin videos on IPFS for us?  I'm thinking one 720p encode and one 360p encode for the IPFS version both using the YouTube encoding spec because it's a pretty reasonable quality / size ratio.  YouTube will get the full 1080p for their [encoding spec](https://support.google.com/youtube/answer/1722171?hl=en) and the Archive.org videos would follow [their respective specifications](https://archive.org/help/derivatives.php).

**Comment by dcwalk** [comment]
@Shrinks99 can you dm me a google-friendly email to add as a manager?

**Comment by lamarqua** [comment]
I think it was mentioned in a conversation I heard at some point that if possible an upload via BitTorrent would be great.

**Comment by dcwalk** [comment]
@Shrinks99 can you update on status? I'd love if we could see some sample footage soonish!!

**Comment by Shrinks99** [comment]
Today I finish my POLS essay and will _finally_ have time to allocate to the video editing!  I did a test earlier of how hard the delay will be to sync up and it's going to be no problem for the vast majority of the videos!  Some videos will be easier to edit than others (talks that require no actual editing work and are just a straight shot all the way through), I expect Jennifer's to be difficult as I need to include the slides in post so there's a lot to do there.  I'd like to get Burak's talk done first as it's already a video and pretty much just needs an intro added to it, after that I plan on doing the lightning talks and then Althea but that's all the scheduling I've done for now.

### Intro!
I've been working on intro music, it's not done but it should be cool.  I'll also be making a short intro clip with the presenter name and talk title, it's going to be pretty simple.

**Comment by dcwalk** [comment]
can we set a deadline for the Our Networks videos being up? We have some final emails to send out and I'd like to lock it down. Next week will be a month since the event-- It'd be great to say by *Friday August 17* if possible, does that work with your schedule?

**Comment by benhylau** [comment]
@Shrinks99 also a final confirmation on _where_ the videos will be uploaded to?

**Comment by Shrinks99** [comment]
I am delivering everything to YouTube as I finish them as unpublished videos.  These will have thumbnails, titles and all the data in the "Advanced Settings" and "Translations" tabs filled out.  I do not know what the best options are for IPFS uploading and would appreciate it if somebody else took care of that.  I will get the videos to whomever that is probably over IPFS because that seems like a nice solution although I don't have a computer to leave on all the time for that.

A note on QC, this involves basically just watching all the talks and checking for errors in editing.  I _could_ do this however it would greatly increase the amount of time each video takes by many hours.  If somebody else would be able to take on this task it would also be greatly appreciated.

YouTube channel assets are in [this PR!](https://github.com/ournetworks/artwork/pull/14)

**Comment by Shrinks99** [comment]
I did compose the music!  I hereby license it with WTFPL so that's covered ;P

I agree, the event name must be more prominent.  The two options for that in the intro are taking out the Our Networks logo entirely or just having it disappear and feature the name before showing the name of the talk and presenter.  Would like @garrying's thoughts as well.

I didn't really intend for that card to be used in the archived video and that would be more work for me.  If you really want it then maybe it's better suited as an end card?

**Comment by Shrinks99** [comment]
# Episode List

I will update this as I go, once videos have been delivered for each platform (after the episode has been set to 'Final') they will be marked with 'Delivered'.  Each episode has been marked with the production code I am using... Hopefully it's not overly confusing.

| Video Name       	| Status        	        | YouTube 	| IPFS 	| D.Tube 	| Internet Archive 	|
|------------------	|---------------	|---------	|------	|--------	|------------------	|
| aether            	| Client Review 	|         	|      	|        	|                  	|
| althea            	|               	|         	|      	|        	|                  	|
| artists-network  	|               	|         	|      	|        	|                  	|
| digital-futures  	|               	|         	|      	|        	|                  	|
| fandom           	|               	|         	|      	|        	|                  	|
| global-assets    	|               	|         	|      	|        	|                  	|
| data-together    	|               	|         	|      	|        	|                  	|
| intro     	                 | Client Review 	|         	|      	|        	|                  	|
| ipfs-stream      	|               	|         	|      	|        	|                  	|
| p2p-mesh-apps    |               	|         	|      	|        	|                  	|
| archiving        	|               	|         	|      	|        	|                  	|
| local-networks   	|               	|         	|      	|        	|                  	|
| moneymesh        	|               	|         	|      	|        	|                  	|
| p2p-centralized  	|               	|         	|      	|        	|                  	|
| stable-networks  	|               	|         	|      	|        	|                  	|
| tactics          	         |               	|         	|      	|        	|                  	|
| wireless-toronto 	|               	|         	|      	|        	|                  	|

**Comment by dcwalk** [comment]
@Shrinks99: are the opening and closing included in that? (I think the intro means the intro sequence for each video?)

Reviewing the tests posted I'd still say my earlier comment holds: I'd like the title of the event to be listed & the year to be more prominent. 

Just copying your thoughts on alternative strategies:
> decide if the intro is a problem (it doesn't actually have the event title IN the video and if that's a problem then I will take the logo out of the intro and replace it with text in Terminal Grotesque)
> Alternate option: instead of fading out the logo gets replaced with the event title and then that fades out
> Also maybe we should swap the title of the event with the logo in the thumbnail as well

I'd favour the 2nd option if I am understanding it correctly.

Good to hear others thoughts

**Comment by Shrinks99** [comment]
I favour the second option for the intro as well, it's less work and then we have everything in there!  Win win.  As for the year I could put it with the event title or make it larger but I don't want it to overshadow either the presenter name or talk title.

When I start editing the keynote I will send an unfinished version to Nasma.  This will have the intro in and the audio synced but it will not cut to better elements and will just be one straight shot through of all the footage we have of the talk.  She will okay, disapprove it, or send notes and I'll either move on or complete the video.

**Comment by garrying** [comment]
Trying to weave the discussion in chat to this thread.
@Shrinks99 are we adding the "Beyond DIY" graphic in place of the "Our Networks" animation? I'm in favour of doing that. If we're keeping the talk title, name, year frame, could we:
- Do without the vignetting
- Keep text the same size for both title and name
- Not sure if `2018` makes sense in this context

Additional thoughts about titling videos:
- Would it make sense to have more descriptive titles like: `{{ title of the presentation }} - {{ presenter(s) }} - Our Networks 2018`, or `Our Networks 2018: {{ title of presentation}} by {{ presenter(s) }}`
- Is it correct to assume we will be adding the presentation abstract to the description of each video?

**Comment by dcwalk** [comment]
Thanks @garrying -- I am inline with your thoughts. @Shrinks99 are you able to post an updated intro from feedback for us to move on?

Also-- maybe a sense of timeline now that you've processed some videos?

**Comment by Shrinks99** [comment]
While I wanted the presenter and conference name in the title originally apparently YouTube has a 100 character limit for titles.  We can do exactly this for the filenames though and I will use whatever naming convention I am asked to use for those.

Boo, I liked the vignette!  I can make it more subtle or take it away completely, your call again.

I would really rather not have the static beyond DIY graphic.  I want to convey the relevant information though.  I will update the intro tonight and send it for review as an unlisted video like last time, it will be posted here when done.  The event tagline will be included.

Where do you want the date in the intro?

**Comment by dcwalk** [comment]
@Shrinks99--

Wait, I thought you were going to use the title frame after the intro
animation...

> Alternate option: instead of fading out the logo gets replaced with the
event title and then that fades out

That is how I interpreted this ☝️, that also resolves the issue of where
the year goes (it is on that title!)


To expand a bit on what I mentioned in chat: we used that graphic
throughout the conference, in the opening and closing remarks and in swag
so it feels important and visually unifying to include in the intro to me

**Comment by benhylau** [comment]
## Video

**Intro:**
* [State of Our Networks Beyond DIY: Do It With Others](garry-please-link) static graphic
* Followed by title card containing `{{ title of presentation }}` and `{{ presenter(s) }}` (drop the animation)
* Music @Shrinks99 composed (drop the end note if it doesn't fit in)

**Recording:** Recorded video with AV sync fixes
**Endcard:** None

## Metadata

**Filename:** `Our_Networks_2018_{{ title_of_presentation }}_by_{{ presenter(s) }}.mp4`
**Title:** `Our Networks 2018: {{ title of presentation }} by {{ presenter(s) }}` (crop `by {{ presenters(s) }}` on ones that are too long)
**YouTube Description:** `{{ presentation abstract }}` + `View on /ipfs/<ipfs-hash>` text linked to `https://ipfs.io/ipfs/<ipfs-hash>`
**Publish:** IPFS and YouTube

**Comment by dcwalk** [comment]
Just adding a comment with the new intro link: https://www.youtube.com/watch?v=ihBFtfDI8RU&feature=youtu.be
b/c I don't think we get notifications when a comment is edited.

**Comment by garrying** [comment]
@Shrinks99 One point about the intro: Could the title graphic linger for a few seconds, maybe 3-4 seconds?, after all the elements appear?

**Comment by dcwalk** [comment]
Agree about a little lingering if possible! Otherwise lgtm!

**Comment by Shrinks99** [comment]
Yeah that can totally be done, it does mean that the title of the talk is on for a shorter amount of time though.  If that sounds good I will make the change and consider it final along with a reupload of the Aether talk.

**Comment by benhylau** [comment]
Thanks @Shrinks99 just a little tho I still wanna be able to read the title!

**Comment by Shrinks99** [comment]
Absolutely, part of the reason I wanted to start out with none of the other graphics on screen was to make it hopefully easier to see the text in the beginning!

**Comment by Shrinks99** [comment]
[Aether has been uploaded with the new length shifted titles](https://www.youtube.com/watch?v=A9zL2csO7t8&feature=youtu.be).  I need to know how to deliver to IPFS.

**Comment by benhylau** [comment]
@Shrinks99 I will sort the details to publish on IPFS tonight / tomorrow morning. If I give you keys to `scp` to a server, then I will manually publish each to IPFS and add hashes to https://github.com/ournetworks/2018/issues/87#issuecomment-412313456 table, is that acceptable?

Regarding Aether, the video looks good to me 👍 for title, are we okay with just `{{ title of presentation }}` as title for all videos, provided they will all be added to a `State of Our Networks 2018 Beyond DIY: Do It With Others` playlist in the end? @garrying @dcwalk

**Comment by Shrinks99** [comment]
They get added to the playlist as they get uploaded, as for IPFS that sounds good to me.

**Comment by benhylau** [comment]
So the 1080p videos are ~3.5 GB for an hour. Do we want to upload both a 720p and 1080p? Two parallel folders in IPFS, basically.

**Comment by garrying** [comment]
> are we okay with just `{{ title of presentation }}` as title for all videos, provided they will all be added to a `State of Our Networks 2018 Beyond DIY: Do It With Others` playlist in the end?

I think we'd want to surface presenter name in the title at a minimum. Are we mentioning speaker name(s) in the description? Maybe something we can follow-up on after everything has been uploaded? Playlist is ok with me. I personally don't prefer them, and would usually opt for a search query like `{{ conf name }} {{ year }}` or presenter name to find videos.

>Do we want to upload both a 720p and 1080p? Two parallel folders in IPFS, basically.

It'd be great if both were avaliable. Two folders seems fine. I would also be okay if everything was in one folder with clear affixed (`*_720p`, `*_1080p`) filenames.

**Comment by Shrinks99** [comment]
YouTube searches based on metadata as well as video titles.  Including that information in the description would be a great idea IMO.  As for IPFS storage all future files uploaded will have either `_720p` or `_1080p` added to them and it would be nice to have them in two seperate directories on the server (`/home/ipfs/our-networks-2018/1080p` and `/home/ipfs/our-networks-2018/720p` respectively.  @benhylau if you could set this up and rename / move the current file in there for Aether it would be much appreciated.

If possible I think it would be best to do a rolling release and publish as videos are completed, this doesn't clog up people's subscription feeds and provides a nice steady stream of content for a while but that's just my opinion.  Would like to hear thoughts on release schedules.

**Comment by benhylau** [comment]
I have pin-added the current videos in the following structure:

```
ipfs@ipfs01:~$ ipfs add -r our-networks-2018
added QmPk33q8z8GHhJoDKXGDGn5CWFu61TzTprgupfGe4VNwho our-networks-2018/1080p/Our_Networks_2018_Aether_Distributing_social_networks_without_distributed_consensus_by_Burak_Nehbit_1080p.mp4
added QmfZaho2FZgz3us8vbyK5ueYtQjMx174CASYLaReXsUP6d our-networks-2018/1080p/Our_Networks_2018_Althea_a_more_distributed_ISP_by_Justin_Kilpatrick_1080p.mp4
added QmQkA2fKAiJYUfxsNjvKpPzTDo5oLzHQQ3JCVN1AxXW9eR our-networks-2018/720p/Our_Networks_2018_Aether_Distributing_social_networks_without_distributed_consensus_by_Burak_Nehbit_720p.mp4
added QmeEixBh2YRdVVgpWZKbqKpUxwRW8PW8TRXKCfk7jwwLVE our-networks-2018/720p/Our_Networks_2018_Althea_a_more_distributed_ISP_by_Justin_Kilpatrick_720p.mp4
added QmW87iHnGbpLDsPrScKEHpEfvPD7hSwyjggrNX7VcRdFeQ our-networks-2018/1080p
added QmRBZHeC9qFAnqtadEobWRDD7Z6A4hafDxt1WpWXRsXfsn our-networks-2018/720p
added QmNYFBuB5cMjRX6izPinNYYjSnPW9PovU76UKwiwwasqVT our-networks-2018

ipfs@ipfs01:~$ ipfs pin add -r QmNYFBuB5cMjRX6izPinNYYjSnPW9PovU76UKwiwwasqVT
pinned QmNYFBuB5cMjRX6izPinNYYjSnPW9PovU76UKwiwwasqVT recursively
```

**Comment by Shrinks99** [comment]
Thoughts on just putting that link in the YouTube description instead of the link for each individual video?

**Comment by Shrinks99** [comment]
Justin's Althea talk is now up for approval, the video has been uploaded to the pitts server as well.  All metadata is filled out except for the description field, I would appreciate it if we could get Aether's description figured out and published so that I have a format to work off of.  Maybe @garrying or @benhylau could make sure that's looking good.

I also asked about publishing to DTube based on existing content served over IPFS and was told it was possible!  If we would like to make videos available on that platform (which is esentially a nice front end for IPFS video anyhow) it would require almost no overhead on my part to do.  I would be happy to create the account and give it a go!
![screen shot 2018-08-19 at 1 42 57 pm](https://user-images.githubusercontent.com/5672810/44311484-6cd3df00-a3b6-11e8-847a-00e1d7198a4f.png)

**Comment by dcwalk** [comment]
@Shrinks99 -- I had researched dtube options a few days ago and given current timelines and that we set our priorities to IPFS and Youtube we should pause on dtube for now. 

The process sounds easy enough to do afterwards in any case. I researched dtube and it looks like they don't have existing way to manage accts across multiple managers & required a steemit registration.

**Comment by Shrinks99** [comment]
Wireless Toronto has been finished, this episode has the worst desync issues with the audio sync changing durring the presentation.  Like the opening there are some parts where the audio cuts out due to sync issues.


Justin Kilpatrick has reviewed the Althea episode and given it the green light.  If you want to take his word for it and publish it that's fine by me, alternatively give it another round of QC.
![screen shot 2018-08-30 at 1 16 20 am](https://user-images.githubusercontent.com/5672810/44830938-5d446980-abf2-11e8-929c-861f52a8295d.png)

**Comment by benhylau** [comment]
@Shrinks99 please provide update on video processing status. We are hitting the revised delivery date and the bulk of videos are not yet available. Communication of current status would be helpful.

**Comment by dcwalk** [comment]
Hi @Shrinks99 thanks for the update, just mentioning the same thing I mentioned in chat:

- I can facilitate emails to those folks abt slides that you mentioned
- however, given that we are 2ish months out, I think if they do not respond in a timely fashion (basically within a day or two) we have to proceed with what we have and finalize the best we can

My goal is that we meet our commitment to finalize and release videos as close to the revised timeline as possible.

**Comment by Shrinks99** [comment]
Emails have been sent out to the last remaining presenters for slides, Protocol Tactics and Holding Data Together are being uploaded to YouTube tonight.

As covered in the chat the plan is to do a daily release schedule, this hopefully gives us the required amount of time to hopefully get everyones slides in so the videos can have the best quality possible!

I have spoke to Jennifer and she has emailed back indicating she will be able to help configure the slide and visual content timing.  I will update with progress on that as it happens.

Uploading to the IPFS server will be done all in one go when I have fast internet at Sheridan, I hope that's okay.  Please let me know what / when to do that and if it is dependant on Jennifer's talk being in the directory, as far as I understand this is true but @benhylau is more qualified to tell me what to do and I'd like clarification on this.

**Comment by benhylau** [comment]
We should aim to have everything in the ipfs server by Friday this week. If Jennifer's talk is not yet ready we should still get the others in, so we can start doing QC. We do need all the talks published to the IPFS archive before we can announce it though.

**Comment by Shrinks99** [comment]
Got it, please QC from YouTube.  I am not posting to other places until the video is approved to reduce possible versioning mixups.

**Comment by benhylau** [comment]
@Shrinks99 Once I review can I directly change the status to `Delivered`?

**Comment by Shrinks99** [comment]
@benhylau Yes, please do that!  Otherwise please flip the episode to Changes Needed.

**Comment by dcwalk** [comment]
Okay so what is the best place for me to grab the videos and upload to a collection on Internet Archive, from the ipfs archive?

**Comment by benhylau** [comment]
We found that some of the new videos are really huge and the 720p sizes are not that much smaller:

```
./1080p:
total 23G
-rwx------ 1 ipfs ipfs 3.2G Aug 17 19:47 Our_Networks_2018_Aether_Distributing_social_networks_without_distributed_consensus_by_Burak_Nehbit_1080p.mp4
-rwx------ 1 ipfs ipfs 2.6G Aug 18 20:51 Our_Networks_2018_Althea_a_more_distributed_ISP_by_Justin_Kilpatrick_1080p.mp4
-rwxr-xr-x 1 ipfs ipfs 798M Sep 14 00:45 Our_Networks_2018_Closing_Remarks_by_Dawn_Walker_and_Yurko_1080p.mp4
-rwxr-xr-x 1 ipfs ipfs 703M Sep 10 22:44 Our_Networks_2018_Holding_Data_Together_by_Brendan_OBrien_1080p.mp4
-rwxr-xr-x 1 ipfs ipfs 995M Sep 11 12:05 Our_Networks_2018_IPFS_Livestreaming_by_Yurko_1080p.mp4
-rwxr-xr-x 1 ipfs ipfs 6.7G Sep 13 20:56 Our_Networks_2018_Money_And_Mesh_Networks_by_Jessica_Marshall_and_Ashoka_Finley_1080p.mp4
-rwxr-xr-x 1 ipfs ipfs 1.1G Sep 14 01:58 Our_Networks_2018_Opening_Remarks_by_Dawn_Walker_1080p.mp4
-rwxr-xr-x 1 ipfs ipfs 781M Sep 11 17:26 Our_Networks_2018_Peer_To_Peer_Applications_On_A_Mesh_Network_by_Benedict_Lau_1080p.mp4
-rwxr-xr-x 1 ipfs ipfs 2.2G Sep 13 22:31 Our_Networks_2018_Protocol_Tactics_by_Lars_Gierth_1080p.mp4
-rwxr-xr-x 1 ipfs ipfs 839M Sep 10 22:24 Our_Networks_2018_Web_Archiving_as_DIY_and_DIWO_by_Emily_Maemura_1080p.mp4
-rwxr-xr-x 1 ipfs ipfs 3.3G Sep 14 00:23 Our_Networks_2018_Wireless_Toronto_by_Gabe_Sawhney_1080p.mp4

./720p:
total 19G
-rwx------ 1 ipfs ipfs 2.0G Aug 18 20:47 Our_Networks_2018_Aether_Distributing_social_networks_without_distributed_consensus_by_Burak_Nehbit_720p.mp4
-rwx------ 1 ipfs ipfs 1.9G Aug 18 20:43 Our_Networks_2018_Althea_a_more_distributed_ISP_by_Justin_Kilpatrick_720p.mp4
-rwxr-xr-x 1 ipfs ipfs 777M Sep 14 00:46 Our_Networks_2018_Closing_Remarks_by_Dawn_Walker_and_Yurko_720p.mp4
-rwxr-xr-x 1 ipfs ipfs 526M Sep 10 22:45 Our_Networks_2018_Holding_Data_Together_by_Brendan_OBrien_720p.mp4
-rwxr-xr-x 1 ipfs ipfs 533M Sep 11 12:03 Our_Networks_2018_IPFS_Livestreaming_by_Yurko_720p.mp4
-rwxr-xr-x 1 ipfs ipfs 6.5G Sep 13 21:03 Our_Networks_2018_Money_And_Mesh_Networks_by_Jessica_Marshall_and_Ashoka_Finley_720p.mp4
-rwxr-xr-x 1 ipfs ipfs 1.1G Sep 14 01:55 Our_Networks_2018_Opening_Remarks_by_Dawn_Walker_720p.mp4
-rwxr-xr-x 1 ipfs ipfs 442M Sep 11 17:21 Our_Networks_2018_Peer_To_Peer_Applications_On_A_Mesh_Network_by_Benedict_Lau_720p.mp4
-rwxr-xr-x 1 ipfs ipfs 1.4G Sep 13 22:23 Our_Networks_2018_Protocol_Tactics_by_Lars_Gierth_720p.mp4
-rwxr-xr-x 1 ipfs ipfs 384M Sep 10 22:18 Our_Networks_2018_Web_Archiving_as_DIY_and_DIWO_by_Emily_Maemura_720p.mp4
-rwxr-xr-x 1 ipfs ipfs 2.9G Sep 14 00:17 Our_Networks_2018_Wireless_Toronto_by_Gabe_Sawhney_720p.mp4
```

We have been targeting a variable video bitrate (16 Mbps) that is twice of what I'd expect from a 1080p HD video (~8 Mbps), for both resolution variants, which explains the similar file sizes. The issue is we used an encoding profile for YouTube which I guess is intended for masters, from which YouTube does transcoding from, so it is at a unreasonably high bitrate (and file size).

I propose two options for IPFS publish:

1. Publish 720p (VBR 3 Mbps) and 1080p (VBR 16 Mbps)
1. Publish 720p (VBR 3 Mbps) and 1080p (VBR 8 Mbps)

In the first options, we need to re-encode all 720p variants. We may be able to batch process this overnight. In the second option, both variants will need to be re-encoded, and also we won't have a super hi-quality archive on IPFS but the 1080p is more consumable.

**Comment by Shrinks99** [comment]
I am in favour of option 1.  Batch processing is no problem for me and will be done in one night, uploading all the videos the the server will be a more time consuming process due to the limitations of my internet speed, hopefully I can get my Ethernet port to work at Sheridan and use their network although it has been having problems authenticating lately.

I can also do 480p variants.

**Comment by Shrinks99** [comment]
@benhylau and I have made progress determining a new encoding standard.  1080p content stays the same and exists for the folks out there who are massive snobs (such as myself) and only watch the highest quality possible content.

... For everyone else We've decided on a 720p VBR 3Mb/s min, 3.2Mb/s max standard which looks decent and delivers good enough quality for the file size.  Deliveries to YouTube remain the same, 1080p deliveries will continue to the IPFS server as they have up to now.  720p deliveries to the IPFS server will take place at the end of the process when all videos have been completed as one big batch upload.

Adobe Media Encoder presets will be made available along with whatever source files people want to be archived at the end of the production process.  Currently the project folder sits at about 87 GB.

**Comment by benhylau** [comment]
@Shrinks99 can we get an update on the table? I believe the only one we are waiting on should be `artists-network`? The other ones should be published right?

**Comment by Shrinks99** [comment]
I am meeting with Jennifer tomorrow to discuss what is needed for the slides, hopefully I can get an ETA from her on that and get it out ASAP.  I am waiting on slides from `local-networks` as their talk is _all_ images and it looks like garbage without them, I still haven't received an email response.  I think I can make `p2p-centralized` look decent enough with some colour correction but slides would be nice to have for that one too.  I haven't received a response for it either.

`fandom` will absolutely be up this weekend.

**Comment by dcwalk** [comment]
What is the deadline @Shrinks99 and @benhylau to publish anyway if you don't hear from `local-networks` and `p2p-centralized`? I thought we set that for Sept 15?

At some point we have to publish

**Comment by Shrinks99** [comment]
@dcwalk I would urge you to exhaust all options for local-networks in order to get their slides before deciding to publish their talk and even possibly not publishing it due to the quality problems.  Jennifer and I have discussed what needs to happen for the slide timing and she will be delivering timing information to me ASAP.  I hope to have her talk finished as soon as I can.

If you or @benhylau would be able to contact the presenters for `local-networks` and `p2p-centralized` I think that might be helpful.  I feel badly about having to pester people for these things but at the same time I want to make a version of their talk they can feel confident about sending people to watch.  At this stage I have not heard anything back from either presenter.  Do you have better contact information than email for them?  I do not know if they have even received mine.

**Comment by dcwalk** [comment]
@Shrinks99 -- in your comment above you say
>  I think I can make p2p-centralized look decent enough with some colour correction but slides would be nice to have for that one too. I haven't received a response for it either.

That doesn't sound like either are a dealbreaker? As we are more than 2 months on from when these talks happened we might not get slides from folks. I do not have additional contact information than what was provided by presenters to us.

**Comment by Shrinks99** [comment]
@dcwalk Yeah, not I'm decently sure all the slides for `p2p-centralized` are just text and as such they _may_ be able to just be colour corrected and it might work out fine.  There are a bunch of on-screen demos in that talk too which will have to undergo the same fix and that will just look as good as I can get it, there won't be much I can do there.  Images are a whole different story due to the compression and the colour problems and it's specifically the `local-networks` talk that is a huge problem because it's almost entirely images on slides.  It will look really bad if released without slide replacement and nobody wants that.  Examples can be provided when I get back to Oakville today.

**Comment by Shrinks99** [comment]
`fandom` is currently being uploaded to YouTube, notes for this one only regard Owen G Parry's section of the talk.  It may seem like the audio cuts off each time the slide changes, it was like this in the original version and that is what happens with the source files.

**Comment by Shrinks99** [comment]
`local-networks` [is uploaded to YouTube](https://youtu.be/-FLJfm6uCDs), @benhylau had some concerns about the quality improvement changes made on this video in order to fix the slide colour problem.  To make QC easy I've documented the changes that didn't only involve colour correction in [this Imgur post](https://imgur.com/a/y0gI9ZY).  Frames labeled "plate" are the original source footage and frames labeled "new" is the final slide.

I still haven't received the slide timings and as it is 10:00 on a Sunday I am not in a position to stay up late to implement them tonight.  I am confident Jennifer will send them in a timely manor and I look forward to publishing both `artists-network` and `p2p-centralized` in a timely manor

D-Tube still hasn't activated my `ournetworks` account creation request (I did not buy one outright and perhaps this would have been a better option).  I have been assured by a member of their staff that they will be able to accomplish this when Steem hard-forks as apparently the new hard fork will resolve the issues they currently have with creating new 'free' accounts.  I put in this request about a week ago when I joined their discord to ask questions and mentioned OurNetworks in that room.  To ensure that name was not taken by some bad person (you know how the internet is) I registered the name myself and I am happy to give the credentials to anyone here.  I would also be happy to set up publishing on that platform (should be as easy as linking the site to our IPFS hashes) after everything is up and finaled on YouTube and the IPFS server.  Should that account be given any Steem I will leave it up to everyone here to decide what to do with it.

I have also started writing a retrospective of all the things that should be done differently next year.  It's a long one and would not serve well posted here, where would you like me to put that?

**Comment by Shrinks99** [comment]
`artists-network` [is up on YouTube](https://youtu.be/_iP1mZ2cceg) meaning all the talks are _finished and delivered_!

I'm almost done with the descriptions for all the episodes (pulling from the conference [program page](https://ournetworks.ca/conference/program/) and using the format as agreed upon in the chat) however I don't have anything for the `opening` and `closing` episodes.  @dcwalk If you'd like to write one (it can be one sentence, I just figure we should have something) that can happen or I can just do a little summary of the conference for the opening description and figure something out for the closing one, your call.

Almost finished!  Once we're 100% ready I will set up the auto-release schedule on YouTube.

Looking forward delivering everything this weekend and having things up on Monday! :)

**Comment by benhylau** [comment]
Thanks @Shrinks99 for uploading all the processed files! I have pin-added the following from our Meta Mesh IPFS server:

```
ipfs@ipfs01:~$ ipfs add -r our-networks-2018
added QmPk33q8z8GHhJoDKXGDGn5CWFu61TzTprgupfGe4VNwho our-networks-2018/1080p/Our_Networks_2018_Aether_Distributing_Social_Networks_Without_Distributed_Consensus_by_Burak_Nehbit_1080p.mp4
added QmfZaho2FZgz3us8vbyK5ueYtQjMx174CASYLaReXsUP6d our-networks-2018/1080p/Our_Networks_2018_Althea_A_More_Distributed_ISP_by_Justin_Kilpatrick_1080p.mp4
added QmRC5i25CJAkKCTq22mPr19yYfykbtMug9NbhCUrseWtkF our-networks-2018/1080p/Our_Networks_2018_Building_Something_Together_With_Local_Networks_by_Cecylia_Bocovich_and_Lauren_Lapidge_1080p.mp4
added QmRVhwmQEGdxeLNUq41exH8GJTXY4ZPV45Ktto8VxJRhi7 our-networks-2018/1080p/Our_Networks_2018_Closing_Remarks_by_Dawn_Walker_and_Yurko_1080p.mp4
added QmWvJ8HbTpaDTYAYtZFNwomyZr21YEwTp1bfQGT7Vxjz8w our-networks-2018/1080p/Our_Networks_2018_Fandom_As_An_Alternative_Network_by_Rea_McNamara_and_Owen_G_Parry_and_Maya_Ben_David_1080p.mp4
added QmXAfHZtPGGskDZPGSQ11eGo3Ac6T72YWNiftgjJXH6qHD our-networks-2018/1080p/Our_Networks_2018_Holding_Data_Together_by_Brendan_OBrien_1080p.mp4
added QmT9aFwHo5Sh197ubcDC6zprNkDdb6hkh1uEf2PNqnDLkF our-networks-2018/1080p/Our_Networks_2018_Live_Streaming_Over_IPFS_by_Yurko_1080p.mp4
added QmdXZXwWQzTv8CHA66HowLcysjsfepVNGr9vcnRTDzTphb our-networks-2018/1080p/Our_Networks_2018_Money_And_Mesh_Networks_by_Jessica_Marshall_and_Ashoka_Finley_1080p.mp4
added QmZJ5ZbJTfhQXM57PSWYy1VsHRDfL5AckjXy6W7RmP89AX our-networks-2018/1080p/Our_Networks_2018_Opening_Remarks_by_Dawn_Walker_1080p.mp4
added QmR68beLDqPARyqDNHyAU4cnV77gmm83xrbEQhaQXXYd2c our-networks-2018/1080p/Our_Networks_2018_Our_Stable_Networks_by_Grant_Gallow_and_Jenny_Ryan_1080p.mp4
added QmbkBmVziTnQyEaCjaZtT8ucth2cwX2VsTFip5YHrebFCS our-networks-2018/1080p/Our_Networks_2018_Peer_To_Peer_Applications_On_A_Mesh_Network_by_Benedict_Lau_1080p.mp4
added QmexHoTeJykKxcs1TuoXSvzqiuQ6tB8n1iKcywFudkmV3E our-networks-2018/1080p/Our_Networks_2018_Protocol_Tactics_by_Lars_Gierth_1080p.mp4
added QmfNNccv1rv1kgJaABPncUvaiXrmLq26BDieS55P1beozD our-networks-2018/1080p/Our_Networks_2018_The_Artists_Network_The_Data_Body_And_The_Social_Body_by_Jennifer_Seaman_Cook_1080p.mp4
added QmVMhYj3QzStSPw6PvPdUW1Xd5FguNnq5Btp9kC2EyrNrd our-networks-2018/1080p/Our_Networks_2018_The_Hybrid_Step_Toward_Migrating_To_P2P_by_Vmon_1080p.mp4
added Qmbn2Jy5Jyw9f4Xcjk3AJKmEVE1xiA4ZMjhpeGvi6LQgVQ our-networks-2018/1080p/Our_Networks_2018_Web_Archiving_as_DIY_and_DIWO_by_Emily_Maemura_1080p.mp4
added QmS1PB42fYmBJbdhyyJXKzhrYKo3sy4XvqUQWJtQJQeqhg our-networks-2018/1080p/Our_Networks_2018_Wireless_Toronto_by_Gabe_Sawhney_1080p.mp4
added QmRvHxC16K1hqeWdmtGNQQ6jnogq9pfmbVo4VSZqeET8dE our-networks-2018/720p/Our_Networks_2018_Aether_Distributing_Social_Networks_Without_Distributed_Consensus_by_Burak_Nehbit_720p.mp4
added QmeDdoqpScv1Pm4zDWMeiAxkhmKnCUevDL7LjbZm2LvttD our-networks-2018/720p/Our_Networks_2018_Althea_A_More_Distributed_ISP_by_Justin_Kilpatrick_720p.mp4
added QmZzHGkci41xWkS6VPGmpEBoKWJzYXZNJGepyrcwvMRYad our-networks-2018/720p/Our_Networks_2018_Building_Something_Together_With_Local_Networks_by_Cecylia_Bocovich_and_Lauren_Lapidge_720p.mp4
added QmRmJip9agFXbckr8ZPFb4Fdwn54Xi9iyWh1NdT1D7cmtS our-networks-2018/720p/Our_Networks_2018_Closing_Remarks_by_Dawn_Walker_and_Yurko_720p.mp4
added Qmdz1mMaC9Urpf7tqsk8TatFwCAwmPrrRD88Zcq2wLF4Ze our-networks-2018/720p/Our_Networks_2018_Fandom_As_An_Alternative_Network_by_Rea_McNamara_and_Owen_G_Parry_and_Maya_Ben_David_720p.mp4
added QmXDLFpTce1A5vuTssrcjwNTFCC8TKqRHkicGy6C3i5PGY our-networks-2018/720p/Our_Networks_2018_Holding_Data_Together_by_Brendan_OBrien_720p.mp4
added QmNtSy8JxmURQzFmW5r44JV6T3KfatLbEFFvePG1a1xK8i our-networks-2018/720p/Our_Networks_2018_Live_Streaming_Over_IPFS_by_Yurko_720p.mp4
added QmSVYASXXnhK3vF4qPraK4jfgBp1ZzMwDXX2xii2FhhrzB our-networks-2018/720p/Our_Networks_2018_Money_And_Mesh_Networks_by_Jessica_Marshall_and_Ashoka_Finley_720.mp4
added QmcCXAbjDwnkhzL8qfrbJLH6DqEbrkBvaF8AiXsdXNLiDL our-networks-2018/720p/Our_Networks_2018_Opening_Remarks_by_Dawn_Walker_720p.mp4
added QmXjZLx83VMnJUQd4KAGZn5uZSpowLvfhsKKcX7qgu46kP our-networks-2018/720p/Our_Networks_2018_Our_Stable_Networks_by_Grant_Gallow_and_Jenny_Ryan_720p.mp4
added QmePW5VKnpM44X2pTcFEgJ8mbTc5bgaYy6qkismZeYkMZo our-networks-2018/720p/Our_Networks_2018_Peer_To_Peer_Applications_On_A_Mesh_Network_by_Benedict_Lau_720p.mp4
added QmRteJr9TSyLoWgki5ZVnYikmBfCrKE2L2XvaY128egGPh our-networks-2018/720p/Our_Networks_2018_Protocol_Tactics_by_Lars_Gierth_720p.mp4
added Qmcbzn27ULvDj2ToLqGVq2UaBNHTaPZ3Ghe524UmKetScj our-networks-2018/720p/Our_Networks_2018_The_Artists_Network_The_Data_Body_And_The_Social_Body_by_Jennifer_Seaman_Cook_720p.mp4
added QmVYB1VFRKfG3ioGL7Lv4FHDRjeygwW59WPXCsBpsxb2Gc our-networks-2018/720p/Our_Networks_2018_The_Hybrid_Step_Toward_Migrating_To_P2P_by_Vmon_720p.mp4
added QmPtKBecxWWNP3frvSi9hUZocpgj8Ea1DWQx6qtFoqvitM our-networks-2018/720p/Our_Networks_2018_Web_Archiving_as_DIY_and_DIWO_by_Emily_Maemura_720p.mp4
added Qme8TaDvDX9eduymzDw5DFdpuqx11xq967PGMNafXawdpM our-networks-2018/720p/Our_Networks_2018_Wireless_Toronto_by_Gabe_Sawhney_720p.mp4
added QmPh8JKAKqKZksD92UnymbZ2rj8FLksCJhkyUSXa7sgZk2 our-networks-2018/1080p
added QmRMiN9wCAvmvkfTcztXrPdd9gTdNZDfzycapfSNtVhkc4 our-networks-2018/720p
added QmeGYzyWAcrX1k38m7gYwfuHxSdr3JzgB4J7KuDUFJG7PX our-networks-2018

ipfs@ipfs01:~$ ipfs pin add QmeGYzyWAcrX1k38m7gYwfuHxSdr3JzgB4J7KuDUFJG7PX
pinned QmeGYzyWAcrX1k38m7gYwfuHxSdr3JzgB4J7KuDUFJG7PX recursively

ipfs@ipfs01:~$ ipfs pin ls QmeGYzyWAcrX1k38m7gYwfuHxSdr3JzgB4J7KuDUFJG7PX
QmeGYzyWAcrX1k38m7gYwfuHxSdr3JzgB4J7KuDUFJG7PX recursive
```

We should be using these `720p` versions for our streams:
https://ipfs.io/ipfs/QmeGYzyWAcrX1k38m7gYwfuHxSdr3JzgB4J7KuDUFJG7PX/720p

@VictorBjelkholm can you pin add archive root `QmeGYzyWAcrX1k38m7gYwfuHxSdr3JzgB4J7KuDUFJG7PX` to ipfs.io and the Guifi gateway server please?

**Comment by benhylau** [comment]
Fixed and pinned, new archive root `Qmc98oo4qdiUQniHjWu2wknaRgPD3JgfHtPvjngcxLLmuY`:

```
ipfs@ipfs01:~$ ipfs add -r our-networks-2018
added QmPk33q8z8GHhJoDKXGDGn5CWFu61TzTprgupfGe4VNwho our-networks-2018/1080p/Our_Networks_2018_Aether_Distributing_Social_Networks_Without_Distributed_Consensus_by_Burak_Nehbit_1080p.mp4
added QmfZaho2FZgz3us8vbyK5ueYtQjMx174CASYLaReXsUP6d our-networks-2018/1080p/Our_Networks_2018_Althea_A_More_Distributed_ISP_by_Justin_Kilpatrick_1080p.mp4
added QmRC5i25CJAkKCTq22mPr19yYfykbtMug9NbhCUrseWtkF our-networks-2018/1080p/Our_Networks_2018_Building_Something_Together_With_Local_Networks_by_Cecylia_Bocovich_and_Lauren_Lapidge_1080p.mp4
added QmRVhwmQEGdxeLNUq41exH8GJTXY4ZPV45Ktto8VxJRhi7 our-networks-2018/1080p/Our_Networks_2018_Closing_Remarks_by_Dawn_Walker_and_Yurko_1080p.mp4
added QmWvJ8HbTpaDTYAYtZFNwomyZr21YEwTp1bfQGT7Vxjz8w our-networks-2018/1080p/Our_Networks_2018_Fandom_As_An_Alternative_Network_by_Rea_McNamara_and_Owen_G_Parry_and_Maya_Ben_David_1080p.mp4
added QmXAfHZtPGGskDZPGSQ11eGo3Ac6T72YWNiftgjJXH6qHD our-networks-2018/1080p/Our_Networks_2018_Holding_Data_Together_by_Brendan_OBrien_1080p.mp4
added QmT9aFwHo5Sh197ubcDC6zprNkDdb6hkh1uEf2PNqnDLkF our-networks-2018/1080p/Our_Networks_2018_Live_Streaming_Over_IPFS_by_Yurko_1080p.mp4
added QmdXZXwWQzTv8CHA66HowLcysjsfepVNGr9vcnRTDzTphb our-networks-2018/1080p/Our_Networks_2018_Money_And_Mesh_Networks_by_Jessica_Marshall_and_Ashoka_Finley_1080p.mp4
added QmZJ5ZbJTfhQXM57PSWYy1VsHRDfL5AckjXy6W7RmP89AX our-networks-2018/1080p/Our_Networks_2018_Opening_Remarks_by_Dawn_Walker_1080p.mp4
added QmR68beLDqPARyqDNHyAU4cnV77gmm83xrbEQhaQXXYd2c our-networks-2018/1080p/Our_Networks_2018_Our_Stable_Networks_by_Grant_Gallo_and_Jenny_Ryan_1080p.mp4
added QmbkBmVziTnQyEaCjaZtT8ucth2cwX2VsTFip5YHrebFCS our-networks-2018/1080p/Our_Networks_2018_Peer_To_Peer_Applications_On_A_Mesh_Network_by_Benedict_Lau_1080p.mp4
added QmexHoTeJykKxcs1TuoXSvzqiuQ6tB8n1iKcywFudkmV3E our-networks-2018/1080p/Our_Networks_2018_Protocol_Tactics_by_Lars_Gierth_1080p.mp4
added QmfNNccv1rv1kgJaABPncUvaiXrmLq26BDieS55P1beozD our-networks-2018/1080p/Our_Networks_2018_The_Artists_Network_The_Data_Body_And_The_Social_Body_by_Jennifer_Seaman_Cook_1080p.mp4
added QmVMhYj3QzStSPw6PvPdUW1Xd5FguNnq5Btp9kC2EyrNrd our-networks-2018/1080p/Our_Networks_2018_The_Hybrid_Step_Toward_Migrating_To_P2P_by_Vmon_1080p.mp4
added Qmbn2Jy5Jyw9f4Xcjk3AJKmEVE1xiA4ZMjhpeGvi6LQgVQ our-networks-2018/1080p/Our_Networks_2018_Web_Archiving_as_DIY_and_DIWO_by_Emily_Maemura_1080p.mp4
added QmS1PB42fYmBJbdhyyJXKzhrYKo3sy4XvqUQWJtQJQeqhg our-networks-2018/1080p/Our_Networks_2018_Wireless_Toronto_by_Gabe_Sawhney_1080p.mp4
added QmRvHxC16K1hqeWdmtGNQQ6jnogq9pfmbVo4VSZqeET8dE our-networks-2018/720p/Our_Networks_2018_Aether_Distributing_Social_Networks_Without_Distributed_Consensus_by_Burak_Nehbit_720p.mp4
added QmeDdoqpScv1Pm4zDWMeiAxkhmKnCUevDL7LjbZm2LvttD our-networks-2018/720p/Our_Networks_2018_Althea_A_More_Distributed_ISP_by_Justin_Kilpatrick_720p.mp4
added QmZzHGkci41xWkS6VPGmpEBoKWJzYXZNJGepyrcwvMRYad our-networks-2018/720p/Our_Networks_2018_Building_Something_Together_With_Local_Networks_by_Cecylia_Bocovich_and_Lauren_Lapidge_720p.mp4
added QmRmJip9agFXbckr8ZPFb4Fdwn54Xi9iyWh1NdT1D7cmtS our-networks-2018/720p/Our_Networks_2018_Closing_Remarks_by_Dawn_Walker_and_Yurko_720p.mp4
added Qmdz1mMaC9Urpf7tqsk8TatFwCAwmPrrRD88Zcq2wLF4Ze our-networks-2018/720p/Our_Networks_2018_Fandom_As_An_Alternative_Network_by_Rea_McNamara_and_Owen_G_Parry_and_Maya_Ben_David_720p.mp4
added QmXDLFpTce1A5vuTssrcjwNTFCC8TKqRHkicGy6C3i5PGY our-networks-2018/720p/Our_Networks_2018_Holding_Data_Together_by_Brendan_OBrien_720p.mp4
added QmNtSy8JxmURQzFmW5r44JV6T3KfatLbEFFvePG1a1xK8i our-networks-2018/720p/Our_Networks_2018_Live_Streaming_Over_IPFS_by_Yurko_720p.mp4
added QmSVYASXXnhK3vF4qPraK4jfgBp1ZzMwDXX2xii2FhhrzB our-networks-2018/720p/Our_Networks_2018_Money_And_Mesh_Networks_by_Jessica_Marshall_and_Ashoka_Finley_720.mp4
added QmcCXAbjDwnkhzL8qfrbJLH6DqEbrkBvaF8AiXsdXNLiDL our-networks-2018/720p/Our_Networks_2018_Opening_Remarks_by_Dawn_Walker_720p.mp4
added QmXjZLx83VMnJUQd4KAGZn5uZSpowLvfhsKKcX7qgu46kP our-networks-2018/720p/Our_Networks_2018_Our_Stable_Networks_by_Grant_Gallo_and_Jenny_Ryan_720p.mp4
added QmePW5VKnpM44X2pTcFEgJ8mbTc5bgaYy6qkismZeYkMZo our-networks-2018/720p/Our_Networks_2018_Peer_To_Peer_Applications_On_A_Mesh_Network_by_Benedict_Lau_720p.mp4
added QmRteJr9TSyLoWgki5ZVnYikmBfCrKE2L2XvaY128egGPh our-networks-2018/720p/Our_Networks_2018_Protocol_Tactics_by_Lars_Gierth_720p.mp4
added Qmcbzn27ULvDj2ToLqGVq2UaBNHTaPZ3Ghe524UmKetScj our-networks-2018/720p/Our_Networks_2018_The_Artists_Network_The_Data_Body_And_The_Social_Body_by_Jennifer_Seaman_Cook_720p.mp4
added QmVYB1VFRKfG3ioGL7Lv4FHDRjeygwW59WPXCsBpsxb2Gc our-networks-2018/720p/Our_Networks_2018_The_Hybrid_Step_Toward_Migrating_To_P2P_by_Vmon_720p.mp4
added QmPtKBecxWWNP3frvSi9hUZocpgj8Ea1DWQx6qtFoqvitM our-networks-2018/720p/Our_Networks_2018_Web_Archiving_as_DIY_and_DIWO_by_Emily_Maemura_720p.mp4
added Qme8TaDvDX9eduymzDw5DFdpuqx11xq967PGMNafXawdpM our-networks-2018/720p/Our_Networks_2018_Wireless_Toronto_by_Gabe_Sawhney_720p.mp4
added QmVaNUwULJCv88U3P2THhgJSgmVxoQs7gNoXETbvULba5S our-networks-2018/1080p
added Qma9jGVeocAuDyns7PEUC8QJnLrdrFjBboQpnFhsGa8zKZ our-networks-2018/720p
added Qmc98oo4qdiUQniHjWu2wknaRgPD3JgfHtPvjngcxLLmuY our-networks-2018

ipfs@ipfs01:~$ ipfs pin add Qmc98oo4qdiUQniHjWu2wknaRgPD3JgfHtPvjngcxLLmuY
pinned Qmc98oo4qdiUQniHjWu2wknaRgPD3JgfHtPvjngcxLLmuY recursively

ipfs@ipfs01:~$ ipfs pin ls Qmc98oo4qdiUQniHjWu2wknaRgPD3JgfHtPvjngcxLLmuY
Qmc98oo4qdiUQniHjWu2wknaRgPD3JgfHtPvjngcxLLmuY recursive
```

**Comment by benhylau** [comment]
New archive again, this time with descriptions and fixed file names `QmQQfD1ztg1aG82VmUmutusbxn7tL4c2o1qHv1ivyWbNj6`:

```
added QmPk33q8z8GHhJoDKXGDGn5CWFu61TzTprgupfGe4VNwho our-networks-2018/1080p/Our_Networks_2018_Aether_Distributing_Social_Networks_Without_Distributed_Consensus_by_Burak_Nehbit_1080p.mp4
added QmfZaho2FZgz3us8vbyK5ueYtQjMx174CASYLaReXsUP6d our-networks-2018/1080p/Our_Networks_2018_Althea_A_More_Distributed_ISP_by_Justin_Kilpatrick_1080p.mp4
added QmRC5i25CJAkKCTq22mPr19yYfykbtMug9NbhCUrseWtkF our-networks-2018/1080p/Our_Networks_2018_Building_Something_Together_With_Local_Networks_by_Cecylia_Bocovich_and_Lauren_Lapidge_1080p.mp4
added QmRVhwmQEGdxeLNUq41exH8GJTXY4ZPV45Ktto8VxJRhi7 our-networks-2018/1080p/Our_Networks_2018_Closing_Remarks_by_Dawn_Walker_and_Yurko_1080p.mp4
added QmWvJ8HbTpaDTYAYtZFNwomyZr21YEwTp1bfQGT7Vxjz8w our-networks-2018/1080p/Our_Networks_2018_Fandom_As_An_Alternative_Network_by_Rea_McNamara_and_Owen_G_Parry_and_Maya_Ben_David_1080p.mp4
added QmXAfHZtPGGskDZPGSQ11eGo3Ac6T72YWNiftgjJXH6qHD our-networks-2018/1080p/Our_Networks_2018_Holding_Data_Together_by_Brendan_OBrien_1080p.mp4
added QmT9aFwHo5Sh197ubcDC6zprNkDdb6hkh1uEf2PNqnDLkF our-networks-2018/1080p/Our_Networks_2018_Live_Streaming_Over_IPFS_by_Yurko_1080p.mp4
added QmdXZXwWQzTv8CHA66HowLcysjsfepVNGr9vcnRTDzTphb our-networks-2018/1080p/Our_Networks_2018_Money_And_Mesh_Networks_by_Jessica_Marshall_and_Ashoka_Finley_1080p.mp4
added QmZJ5ZbJTfhQXM57PSWYy1VsHRDfL5AckjXy6W7RmP89AX our-networks-2018/1080p/Our_Networks_2018_Opening_Remarks_by_Dawn_Walker_1080p.mp4
added QmR68beLDqPARyqDNHyAU4cnV77gmm83xrbEQhaQXXYd2c our-networks-2018/1080p/Our_Networks_2018_Our_Stable_Networks_by_Grant_Gallo_and_Jenny_Ryan_1080p.mp4
added QmbkBmVziTnQyEaCjaZtT8ucth2cwX2VsTFip5YHrebFCS our-networks-2018/1080p/Our_Networks_2018_Peer_To_Peer_Applications_On_A_Mesh_Network_by_Benedict_Lau_1080p.mp4
added QmexHoTeJykKxcs1TuoXSvzqiuQ6tB8n1iKcywFudkmV3E our-networks-2018/1080p/Our_Networks_2018_Protocol_Tactics_by_Lars_Gierth_1080p.mp4
added QmfNNccv1rv1kgJaABPncUvaiXrmLq26BDieS55P1beozD our-networks-2018/1080p/Our_Networks_2018_The_Artists_Network_The_Data_Body_And_The_Social_Body_by_Jennifer_Seaman_Cook_1080p.mp4
added QmVMhYj3QzStSPw6PvPdUW1Xd5FguNnq5Btp9kC2EyrNrd our-networks-2018/1080p/Our_Networks_2018_The_Hybrid_Step_Toward_Migrating_To_P2P_by_Vmon_1080p.mp4
added Qmbn2Jy5Jyw9f4Xcjk3AJKmEVE1xiA4ZMjhpeGvi6LQgVQ our-networks-2018/1080p/Our_Networks_2018_Web_Archiving_As_DIY_And_DIWO_by_Emily_Maemura_1080p.mp4
added QmS1PB42fYmBJbdhyyJXKzhrYKo3sy4XvqUQWJtQJQeqhg our-networks-2018/1080p/Our_Networks_2018_Wireless_Toronto_by_Gabe_Sawhney_1080p.mp4
added QmRvHxC16K1hqeWdmtGNQQ6jnogq9pfmbVo4VSZqeET8dE our-networks-2018/720p/Our_Networks_2018_Aether_Distributing_Social_Networks_Without_Distributed_Consensus_by_Burak_Nehbit_720p.mp4
added QmeDdoqpScv1Pm4zDWMeiAxkhmKnCUevDL7LjbZm2LvttD our-networks-2018/720p/Our_Networks_2018_Althea_A_More_Distributed_ISP_by_Justin_Kilpatrick_720p.mp4
added QmZzHGkci41xWkS6VPGmpEBoKWJzYXZNJGepyrcwvMRYad our-networks-2018/720p/Our_Networks_2018_Building_Something_Together_With_Local_Networks_by_Cecylia_Bocovich_and_Lauren_Lapidge_720p.mp4
added QmRmJip9agFXbckr8ZPFb4Fdwn54Xi9iyWh1NdT1D7cmtS our-networks-2018/720p/Our_Networks_2018_Closing_Remarks_by_Dawn_Walker_and_Yurko_720p.mp4
added Qmdz1mMaC9Urpf7tqsk8TatFwCAwmPrrRD88Zcq2wLF4Ze our-networks-2018/720p/Our_Networks_2018_Fandom_As_An_Alternative_Network_by_Rea_McNamara_and_Owen_G_Parry_and_Maya_Ben_David_720p.mp4
added QmXDLFpTce1A5vuTssrcjwNTFCC8TKqRHkicGy6C3i5PGY our-networks-2018/720p/Our_Networks_2018_Holding_Data_Together_by_Brendan_OBrien_720p.mp4
added QmNtSy8JxmURQzFmW5r44JV6T3KfatLbEFFvePG1a1xK8i our-networks-2018/720p/Our_Networks_2018_Live_Streaming_Over_IPFS_by_Yurko_720p.mp4
added QmSVYASXXnhK3vF4qPraK4jfgBp1ZzMwDXX2xii2FhhrzB our-networks-2018/720p/Our_Networks_2018_Money_And_Mesh_Networks_by_Jessica_Marshall_and_Ashoka_Finley_720p.mp4
added QmcCXAbjDwnkhzL8qfrbJLH6DqEbrkBvaF8AiXsdXNLiDL our-networks-2018/720p/Our_Networks_2018_Opening_Remarks_by_Dawn_Walker_720p.mp4
added QmXjZLx83VMnJUQd4KAGZn5uZSpowLvfhsKKcX7qgu46kP our-networks-2018/720p/Our_Networks_2018_Our_Stable_Networks_by_Grant_Gallo_and_Jenny_Ryan_720p.mp4
added QmePW5VKnpM44X2pTcFEgJ8mbTc5bgaYy6qkismZeYkMZo our-networks-2018/720p/Our_Networks_2018_Peer_To_Peer_Applications_On_A_Mesh_Network_by_Benedict_Lau_720p.mp4
added QmRteJr9TSyLoWgki5ZVnYikmBfCrKE2L2XvaY128egGPh our-networks-2018/720p/Our_Networks_2018_Protocol_Tactics_by_Lars_Gierth_720p.mp4
added Qmcbzn27ULvDj2ToLqGVq2UaBNHTaPZ3Ghe524UmKetScj our-networks-2018/720p/Our_Networks_2018_The_Artists_Network_The_Data_Body_And_The_Social_Body_by_Jennifer_Seaman_Cook_720p.mp4
added QmVYB1VFRKfG3ioGL7Lv4FHDRjeygwW59WPXCsBpsxb2Gc our-networks-2018/720p/Our_Networks_2018_The_Hybrid_Step_Toward_Migrating_To_P2P_by_Vmon_720p.mp4
added QmPtKBecxWWNP3frvSi9hUZocpgj8Ea1DWQx6qtFoqvitM our-networks-2018/720p/Our_Networks_2018_Web_Archiving_As_DIY_And_DIWO_by_Emily_Maemura_720p.mp4
added Qme8TaDvDX9eduymzDw5DFdpuqx11xq967PGMNafXawdpM our-networks-2018/720p/Our_Networks_2018_Wireless_Toronto_by_Gabe_Sawhney_720p.mp4
added QmQ2osCcQmSz5gj3JEBVcv2mhnkFhWPEWLEmwMesArEGdM our-networks-2018/descriptions/Our_Networks_2018_Aether_Distributing_Social_Networks_Without_Distributed_Consensus_by_Burak_Nehbit.txt
added QmfMmD6JmJPTvK16oJEDc5b8mzJyLKyuoreTVAf3SLYVRk our-networks-2018/descriptions/Our_Networks_2018_Althea_A_More_Distributed_ISP_by_Justin_Kilpatrick.txt
added QmUnwfoxS8w4q8zERJjEZiE2Ct6xFW63GeK61nDMtSWbwy our-networks-2018/descriptions/Our_Networks_2018_Building_Something_Together_With_Local_Networks_by_Cecylia_Bocovich_and_Lauren_Lapidge.txt
added QmS48kpaFmoFicj1C5tLey4JaoTdKimseofT1t15RkRPnk our-networks-2018/descriptions/Our_Networks_2018_Closing_Remarks_by_Dawn_Walker_and_Yurko.txt
added QmcnkFTwfNcyFteDSgpBoyh5gLWNo3gEcnUvr8X4nF1LDj our-networks-2018/descriptions/Our_Networks_2018_Fandom_As_An_Alternative_Network_by_Rea_McNamara_and_Owen_G_Parry_and_Maya_Ben_David.txt
added QmXjwMHBEBwRN3nsiaB1mF6QubdkPZhb4ToBYAFdYuYdma our-networks-2018/descriptions/Our_Networks_2018_Holding_Data_Together_by_Brendan_OBrien.txt
added QmXo1fM3MGxZy3UuzAKauz65nq4PqZ4HrUt75m5oqSnYHd our-networks-2018/descriptions/Our_Networks_2018_Live_Streaming_Over_IPFS_by_Yurko.txt
added QmPYwPYjM7Qt5MADMFeL3fU8UzoVzHc1j2HNddLf8o647h our-networks-2018/descriptions/Our_Networks_2018_Money_And_Mesh_Networks_by_Jessica_Marshall_and_Ashoka_Finley.txt
added QmaeRhXZexESLZggYxNw4fJGfrh88cxUQhXV3kCF56wEyJ our-networks-2018/descriptions/Our_Networks_2018_Opening_Remarks_by_Dawn_Walker.txt
added QmXUqJRsE8v2EoKsH4N3jXJSYc4xbxwCGwuaA3knvTQGCW our-networks-2018/descriptions/Our_Networks_2018_Our_Stable_Networks_by_Grant_Gallo_and_Jenny_Ryan.txt
added QmSi1hEuHRTcePbyZh8caP7nSzepFN8JtmWmBXSmR2pARy our-networks-2018/descriptions/Our_Networks_2018_Peer_To_Peer_Applications_On_A_Mesh_Network_by_Benedict_Lau.txt
added QmSi7jtDgToppNNwFuq42W7gHX4o2zeao2J6QqNFmpArN5 our-networks-2018/descriptions/Our_Networks_2018_Protocol_Tactics_by_Lars_Gierth.txt
added QmXAkfyGW5mg5aDqgmU9Qyu1BDPEuYawch5FXJcGYKabWv our-networks-2018/descriptions/Our_Networks_2018_The_Artists_Network_The_Data_Body_And_The_Social_Body_by_Jennifer_Seaman_Cook.txt
added QmZGCu2QCsP3hFGnnRxsadq7bEpBxkvUQ45ia1oYmCA3Ri our-networks-2018/descriptions/Our_Networks_2018_The_Hybrid_Step_Toward_Migrating_To_P2P_by_Vmon.txt
added QmfCz29hA73AtpoDeEYRxLPkLY31Tcej4nD88kCoUeqqYL our-networks-2018/descriptions/Our_Networks_2018_Web_Archiving_As_DIY_And_DIWO_by_Emily_Maemura.txt
added Qmd3GVgsrsDJPJLmMcvSbD5WSLxn8QXSuDd7AmXs3gT8xN our-networks-2018/descriptions/Our_Networks_2018_Wireless_Toronto_by_Gabe_Sawhney.txt
added QmPvsUh5vijsn7JBmy3rGHYFWuAXGwDyt8vnTFLVEcyBeb our-networks-2018/1080p
added QmQwRjRntgVo2nquaDLTNgaRJsNnh9h2gEbBWiWKnH98M3 our-networks-2018/720p
added QmXEt538SA54Nv9VmyZB5RgPA1ZPoFuLAFveyACXCqLVDD our-networks-2018/descriptions
added QmQQfD1ztg1aG82VmUmutusbxn7tL4c2o1qHv1ivyWbNj6 our-networks-2018
```

**Comment by Shrinks99** [comment]
[YouTube videos have all been made public](https://www.youtube.com/channel/UCudGRFTjVGsrKF0h27GER2g) ahead of Monday's release.

**Comment by Shrinks99** [comment]
Are we still linking to the other platforms in the YouTube description?  If so, what should be there?

**Comment by dcwalk** [comment]
Okay uploaded at Internet Archive: https://archive.org/details/ournetworks2018, will PR against the talks page shortly

**Comment by dcwalk** [comment]
This is done! Youtube descriptions are updated. Mega issue closed :door:
